client/app/scripts/superdesk-desks/views/stage-item-list.html

```diff
@@ -14,8 +14,8 @@
 					<span class="headline">{{item.headline}}</span>
 				</div>
 				<div class="action" ng-if="allowed">
-					<button ng-click="preview(item); $event.stopPropagation();" tooltip="Preview" tooltip-placement="bottom"><i class="icon-external"></i></button>
-			 		<button ng-click="edit(item); $event.stopPropagation();" tooltip="Edit" tooltip-placement="bottom"><i class="icon-pencil"></i></button>
+					<button ng-click="preview(item); $event.stopPropagation();" tooltip="Preview" tooltip-placement="{{$last && 'left' || 'bottom'}}"><i class="icon-external"></i></button>
+			 		<button ng-click="edit(item); $event.stopPropagation();" tooltip="Edit" tooltip-placement="{{$last && 'left' || 'bottom'}}"><i class="icon-pencil"></i></button>

```

Comment:
Perhaps using a ternary operator would make this expression a bit more readable.
```javascript
$last ? 'left' :'bottom'
```